### PR TITLE
FEATURE: Search items NOT described with a given topic (closes #120)

### LINKED ISSUE #120: Search items NOT described with a given topic
## Content

Be able to do crossed search including negative requests (select several topics and display all item photos that match that topic). 

---

## Checklist

- [x] Scenarios ( @timothee-bayart & @LeafStark )
- [x] Mock-up (@jbaptisteblot &, @acomteutt)
- [x] Automated tests ( @timothee-bayart & @LeafStark )
- [x] Implementation (@aclairet & @vcarrara & @jbaptisteblot )

### PR DESCRIPTION
**Pull request by jbaptisteblot**
## Content

implements #120 with associated test.

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [x] corresponds to a contribution that should be notified to users,
    - [x] does not generate new errors or warnings at compile or test time,
    - [x] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [x] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [x] be followed by a colon (`:`) with one space after and no space before,
    - [x] be followed by a title (written in English) as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [x] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [ ] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [ ] related to this very contribution (feature, fix...),
    - [ ] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [ ] without any typo in variable, class or function names,
    - [ ] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

File: features/step_definitions/portfolio.rb [hunk 1]
```diff
@@ -110,6 +113,9 @@ def getUUID(itemName)
   click_on item
 end
 
+Quand("l'utilisateur exclue la rubrique {string}") do |topic|
+  click_on topic
```
Comment (benel) [inline features/step_definitions/portfolio.rb:117]: On exclue une rubrique en cliquant sur une rubrique sélectionnée, c'est ça ?
Du coup quelle est la différence entre le fait de désélectionner ? Le nombre de clics ?

Comment (jbaptisteblot) [reply features/step_definitions/portfolio.rb:117]: C'est bien ça, pour désélectionner une catégorie il faut cliquer une fois supplémentaire

Comment (benel) [reply features/step_definitions/portfolio.rb:117]: Attention, il y a une petite incohérence entre l'instance de l'étape (`Quand on exclue`) et sa définition (`Quand l'utilisateur exclue`). Du coup, l'exécution des tests indique le scénario comme `undefined`.

**Comment by benel** [comment]
Thanks for this fix. Could you please rewrite the history with only two commits?
Yes I know ;) But then I will be sure that Git is no more a problem for you.
Note: You will need `--force` when you `push` since you rewrite the history.

**Comment by benel** [comment]
Please also note that `closes` should be used only on the FEATURE (not the TEST).
You cannot close a ticket twice...

**Review by benel — COMMENTED** [review]
Thanks for your contribution.

**Review by benel — APPROVED** [review]
Approved for Integration.
Thanks for your contribution!

**Pull request merged**